ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-isis-new.yml

```diff
@@ -62,6 +62,52 @@ router_isis:
         index: 211
       - prefix: 2001:cafe:155::/64
         index: 6211
+  set_overload_bit:
+    on_startup:
+      wait_for_bgp:
+        enabled: true
+        timeout: 10
+  authentication:
+    both:
+      key_ids:
+        - id: 2
+          algorithm: sha-512
+          key_type: 0
+          key: password
+          rfc_5310: false
+    level_1:
+      key_type: 0
+      key: password
+      mode: md5
+      rx_disabled: true
```

Comment:
You have set rx_disabled: true here, but i can't see rx-disabled showing up in the generated cli.

Reply:
After reading the template a bit more thoroughly i see it's because the specific level_1/level_2 stuff is only rendered if both is not defined. This looks like intended behavior, so all good.